### DISCUSSION TITLE
Is there a "Raspberry Pi 500" (all-in-one keyboard computer) on the roadmap and will it be based on a SoC derivative of Raspberry Pi 5 / Compute Module 5?

### DISCUSSION THREAD

**Discussion by Hedda** [geerlingguy/raspberry-pi-pcie-devices#689] [General]
Are there any news of "Raspberry Pi 500" (all-in-one [keyboard computer](https://en.wikipedia.org/wiki/Keyboard_computer)) on the roadmap as an updated variant of [Raspberry Pi 400](https://www.raspberrypi.com/products/raspberry-pi-400/)? 

Since the release of the [Raspberry Pi Compute Module 5 (CM5)](https://www.raspberrypi.com/news/compute-module-5-on-sale-now/) there have been some informal Raspberry Pi 500 might just use a CM5:

- https://www.tomshardware.com/raspberry-pi/the-raspberry-pi-500-may-be-right-around-the-corner
- https://hackaday.com/2024/10/04/the-raspberry-pi-500-hints-at-its-existence/
- https://www.notebookcheck.net/Raspberry-Pi-500-makes-first-appearance-as-Raspberry-Pi-400-successor-with-Broadcom-BCM-2712-SoC.898772.0.html
- https://www.raspberrypi.com/news/explaining-computers-beat-us-to-raspberry-pi-500/

So anyone here heard any rumour about a "Raspberry Pi 500" all-in-one keyboard computer and if it will be based on CM5 or not?

For timeline reference:

- Raspberry Pi 500 has not yet been announced, and no one at Raspberry Pi has yet said they will release such a model for sure?

- Raspberry Pi Compute Module 5 (CM5) was announced in November 2024.

- Raspberry Pi 5 was announced in September 2023.

- Raspberry Pi 400 Kit was released in November 2020.

- Raspberry Pi 4 was released in June 2019.

![image](https://github.com/user-attachments/assets/0327d6a7-2a88-4835-9e39-c8aa7cb5db77)

PS: I personally think such as device would sell well and cool that the name "Raspberry Pi 500" reminds me of to the [Amiga 500](https://wikipedia.org/wiki/Amiga_500) (from [Commodore](https://wikipedia.org/wiki/Commodore)) and its predecessor the Commodore 64 (also known as the C64) which was an all-in-one keyboard computer that was hugely popular in the 1980's and early 1990's before iD Software released the first [Doom video game](https://en.wikipedia.org/wiki/Doom_(1993_video_game)) was released for Intel-x86 compatible PCs in 1993 which became the death sentence for non-x86 computers, (and I think interestingly the Amiga series still having a relativly large active fanbas and retro community to this date, especially among European developers who grew up having one in their teens).

**Comment by geerlingguy** [comment]
Unfortunately, this repository isn't a good place for speculation, I'd leave that to other forums.
